src/cls/otp/cls_otp_client.h

```diff
@@ -33,6 +33,10 @@ namespace rados {
         static int get_current_time(librados::IoCtx& ioctx, const std::string& oid,
                                     ceph::real_time *result);
 #endif
+	static void check(CephContext *cct, librados::ObjectReadOperation *op, const std::string& oid,
```

Comment:
if you added new APIs in the header, you need to add them in the cc file (don't replace the existing sync APIs)